### PR TITLE
Rename enabled only filter to include disabled, turn off by default

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
image

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/components/ItemFilterToggle.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default defineComponent({
       return router.push({
         query: {
           ...omit(unref(currentRoute).query, [queryParam]),
-          [queryParam]: unref(filterActive).toString()
+          ...(unref(filterActive) && { [queryParam]: 'true' })
```
Comment (AlexAndBear) [inline packages/web-pkg/src/components/ItemFilterToggle.vue:42]: reverted to original state

**Review by kulmann — CHANGES_REQUESTED** [review]
Could you please adjust the existing changelog item and add the link of this PR to the link list?

**Comment by AlexAndBear** [comment]
> Could you please adjust the existing changelog item and add the link of this PR to the link list?

I did that, but then reverted, since this is in a release folder now, should I still change it, or add a new one ?

**Comment by kulmann** [comment]
> > Could you please adjust the existing changelog item and add the link of this PR to the link list?
> 
> I did that, but then reverted, since this is in a release folder now, should I still change it, or add a new one ?
> 
> See [7de92ee](https://github.com/owncloud/web/commit/7de92ee4bf41c3d3db2d87c2c7049f95c1ab15bb)

Just do that in the release folder. It's from an alpha, doesn't matter. Only would be relevant if it would have been a final release.

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**